.travis.yml

```diff
@@ -1,9 +1,19 @@
+# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 os:
-    - linux
-    - osx
+  - linux
+  - osx
 julia:
-    - 0.5
-    - nightly
+  - 0.7
+  - nightly
+# matrix:
+#   allow_failures:
+#     - julia: nightly
+
 notifications:
-    email: false
+  email: false
+after_success:
+  # push coverage results to Coveralls
+  - julia -e 'cd(Pkg.dir("MeshIO")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
```

Comment:
I assume you've copied this from MeshIO? :wink:

Reply:
Whoops :). Just grabbed a random package's config and forgot to update